changelogs/fragments/cinek810-000001-icinga2_host-fix-use_proxy.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- icinga2_host - fixed the issue with not working use_proxy option of the module
```

You can use restructured text syntax:
```suggestion
- icinga2_host - fixed the issue with not working ``use_proxy`` option of the module.
```

---

thanks for prompt reply!